src/Http/RouteCollection.php

```diff
@@ -73,34 +73,32 @@ public function delete($path, $name, $handler)
 
     public function addRoute($method, $path, $name, $handler)
```

Comment:
Perhaps we should mark all these methods as internal? Or even the entire class?

Reply:
Yeah, the whole class shouldn't be considered public API. We have extenders for this.